src/lighteval/tasks/templates/formatting_utils.py

```diff
@@ -0,0 +1,48 @@
+from lighteval.tasks.templates.translation_literals import TranslationLiterals
+
+
+def decapitalize(word: str):
+    if len(word) == 0:
+        return word
+    return word[0].lower() + word[1:]
+
+
+def capitalize(word: str):
+    if len(word) == 0:
+        return word
+    return word[0].upper() + word[1:]
+
+
+def fix_ending_punct(ctx: str, translation_literals: TranslationLiterals):
+    ctx = ctx.strip()
```

Comment:
maybe we want it to be rstrip instead since we are focusing on the end of the string ?

Reply:
thanks